src/Workspaces/Core/Portable/Execution/SolutionInfoCreator.cs

```diff
@@ -41,7 +41,8 @@ public static async Task<(SolutionInfo, SerializableOptionSet)> CreateSolutionIn
                 }
             }
 
-            var info = SolutionInfo.Create(solutionInfo.Id, solutionInfo.Version, solutionInfo.FilePath, projects);
+            var info = SolutionInfo.Create(solutionInfo.Id, solutionInfo.Version, solutionInfo.FilePath, projects)
```

Comment:
Do we not have a SolutionInfo.WithProjects() or something? This is a good catch but it's terrifying to me that we have to remember to update this with any further new attributes.

Reply:
`solutionInfo` is actually a `SolutionInfo.SolutionAttributes` here